Documentation/Blazorise.Docs/Pages/News/2024-10-15-release-notes-170.razor

```diff
@@ -229,4 +229,12 @@
     However, since this module is not officially supported, we are marking this feature as experimental. This designation will remain until an official table module is released by the Quill.js team.
 </Paragraph>
 
+<Heading Size="HeadingSize.Is3">
+    ToastProvider Improvements
+</Heading>
+
+<Paragraph>
+    We have made some improvements to the ToastProvider component. The ToastProvider is a powerful helper component built on top of the Toast component and is used for showing simple alerts and notifications. The ToastProvider now includes a new option to to handle Opening and Closing events. This can be useful if you want to perform some actions before the toast is shown or hidden.
```

Comment:
Why are we re-explaining what ToastProvider is? This is release notes, the component already exists, I'd just go straight to the point?

Reply:
Because there are always other features going into release notes, I always refine the release notes before the final version. So that the reading feels more nature. This is just so I do not forget it until then.